modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/NodAndShuffleProgress.scala

```diff
@@ -54,20 +54,21 @@ object NodAndShuffleProgressMessage extends ProgressLabel {
       <.div(
         SeqexecStyles.specialStateLabel,
         SeqexecStyles.progressMessage,
-        p.nsStatus.state.map[VdomElement] { nsState =>
+        p.nsStatus.state.map[VdomElement] { _ =>
           s.progressConnect { proxy =>
+            val nsProgress = proxy()

```

Comment:
The use of proxy has been problematic in the past. I prefer to extract the content of the proxy when the component is created and stor it in the props

Reply:
The proxy is provided by the `connect`. The `connect` is defined in the `Props`, but we cannot extract the value outside the `connect` block.